flashcards/web/src/router.tsx

```diff
@@ -10,4 +11,8 @@ export const router = createBrowserRouter([
         path: "/login",
         element: <h1>Login</h1>,
     },
+    {
+        path: "/add-deck",
+        element: <AddDeckTemp />,
```

Comment:
<img width="842" alt="Screenshot 2023-04-09 at 21 33 02" src="https://user-images.githubusercontent.com/82170337/230792878-2d529989-a147-4521-a35e-d9cb8d057d98.png">
When opened on small screen image picker broke down

Reply:
Tab occupies entire screen, close button is partially out of the view port